metricbeat/docs/modules/aws.asciidoc

```diff
@@ -179,6 +181,8 @@ metricbeat.modules:
         - name: InstanceId
           value: i-0686946e22cf9494a
     - namespace: AWS/EBS
+    - namespace: AWS/ELB
+      tags.resource_type_filter: elasticloadbalancing
   #regions:
   #  - us-east-1
   #  - us-east-2
```

Comment:
We may want to split this example in smaller examples for specific use cases, but let's do it in a future PR.

Reply:
Yeah, this is actually generated from `x-pack/metricbeat/module/aws/_meta/config.yml`. Maybe I should keep this config.yml short and then put metricset specific config into for example `x-pack/metricbeat/module/aws/cloudwatch/_meta/docs.asciidoc` so it's easier to read?

Reply:
Yes, we could keep config.yml short, and add different examples to the documentation.